client/flutter/lib/components/home_page_sections/home_page_information_card.dart

```diff
@@ -64,8 +65,10 @@ class HomePageInformationCard extends StatelessWidget {
                       ),
                     ),
                     onPressed: () {
-                      return Navigator.of(context, rootNavigator: true)
-                          .pushNamed(this.link.route, arguments: link.args);
+                      return this.link.isExternal
```

Comment:
Think you should make these 4 lines just a method on RouteLink or a new AppNavigator class as we re-use this in many places.